README.md

```diff
@@ -168,6 +170,9 @@ Emerging ideas are fully supported and are the future for the company business.
 
 Now that you’ve read all about our company purpose, mission and values, and know a bit more about how mondora is organised and functions, you may be asking yourself how these things all fit together. How do we take care of people, of the local community, and of the land in day-by-day activities whilst also building great software and engaging with our customers? Here’s how it all works, and how you can take part in the benefit creating activities that define mondora as a [New Normal company!](https://bcalmbcorp.com/some-old-normal-company-oddities-255a84202552)
 
+## Impact Development
+The impact we are creating is within every action we do. As part of a system, we all agree that we have to take care of all the possibilities we can work out to build positive impact. A Slack channel #insights is dedicated to all the things we encounter in our professional life and where we think we can produce more. You are required to suggest all the things you want the company to follow up that can be useful for every stakeholder. We have some colleagues who have a specific hat of Impact Developers and who are willing to go further (for the world) with the insight.
```

Comment:
We create impact with our every action within the company. As part of a system, we all agree to take care of all the chances we have to build positive impact. We use the Slack channel #insights to share any interesting opportunities that may arise in which we believe there is the possibility of creating positive impact. Here you can suggest opportunities that you would like the company to follow up and that can be useful for our stakeholders. We have colleagues working specifically as Impact Developers and who are willing to go further (for the world) with the insight.